src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -103,6 +103,10 @@ protected function describeContainerServices(ContainerBuilder $builder, array $o
                 continue;
             }
 
+            if (!$showHidden && $builder->hasDefinition($serviceId) && $builder->getDefinition($serviceId)->hasTag('container.excluded')) {
```

Comment:
Use the existing `$service` variable instead of calling `$builder->getDefinition($serviceId).

Reply:
Thank you for your help! Fixed it.